src/EnergyPlus/FaultsManager.cc

```diff
@@ -147,6 +147,8 @@ namespace FaultsManager {
     int const iFault_Fouling_Chiller(115);
     int const iFault_Fouling_EvapCooler(116);
 
+    bool GetFaultsInputFlag(true); // Flag set to make sure you get input once
```

Comment:
Does this need to be added to a clear_state function for unit testing?

Reply:
It should have been included. Will you add it or do you want me to do it?

Reply:
I could but it would be tomorrow before I can get to it.  If you can squeeze it in and also open up a new issue to track the remaining fault warning/behavior then this could merge in.

Reply:
Sure. Will do.

Reply:
@Myoldmopar It is already added in clear_state. May be I do not need to add this variable in the first place because there is one already but it cannot be used outside this module. I am testing if I can use the existing one.